### ISSUE TITLE
[aus prod] Unable to create new product with image

### ISSUE THREAD

**Issue by sstead** [openfoodfoundation/openfoodnetwork#1387]
On production...
If a new product is created without an uploading an image, it works fine. 
If I create a new product, but upload an image I get the snail page.

Could be related to recent push - https://github.com/openfoodfoundation/openfoodnetwork/pull/1331

**Comment by mkllnk** [comment]
Our production server ran out of memory: https://app.bugsnag.com/yaycode/openfoodnetwork/errors/57e1f7c49a8502cbc9b347c8?event_id=587be55f31e910110027fd92

The worker count was recently increased from 2 to 3. I lowered it down to 2 again.

I identified two more ways to increase the available memory:

- Use [posix-spawn](https://github.com/thoughtbot/cocaine#posix-spawn).
- Reduce the amount of running postgres versions. Currently 9.3, 9.5 and 9.6 are running simultaneously. Following https://blog.chaps.io/2016/02/08/upgrading-postgresql-from-9-4-to-9-5-on-ubuntu-15-10.html we should perform the following steps at a quiet time:
```sh
pg_dumpall > 9-4-backup
apt-get remove postgresql-9.5
pg_dropcluster 9.6 main
service unicorn_openfoodweb stop
pg_upgradecluster 9.4 main
service postgresql-9.3 stop
sed -i 's/^port = 5434   /port = 5432   /g'
service postgresql-9.3 restart
service unicorn_openfoodweb start
```

This should be tested on staging first.

I would leave this issue open until the postgres migration has been done.

**Comment by mkllnk** [comment]
The best maintenance window is between 11pm and 6am.

```
Australian production server (2016-12-18 - 2017-01-17)
Hour	Hits
0	69
1	181
2	51
3	29
4	29
5	29
6	91
7	239
8	358
9	448
10	351
11	469
12	490
13	506
14	397
15	441
16	302
17	301
18	247
19	313
20	317
21	331
22	315
23	123
```

**Comment by mkllnk** [comment]
Looking at the staging server I found that they were still running postgres 9.1 and never installed a newer version. Looking at production again, I discovered a 3rd party repository that provides postgres. So I decided to continue using postgres 9.3 while uninstalling 9.5 and 9.6. So we save a bit of memory and CPU here while not changing the current setup. We had in mind to replace the production server at some point. According to https://wiki.ubuntu.com/Releases the current Ubuntu 12.4.5 will not be supported after April 2017.
